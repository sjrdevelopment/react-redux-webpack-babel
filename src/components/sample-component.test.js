import React from 'react'
import renderer from 'react-test-renderer'

import SampleComponent from './sample-component'

test('Sample component renders without errors', () => {
  const component = renderer.create(
    <SampleComponent />
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
