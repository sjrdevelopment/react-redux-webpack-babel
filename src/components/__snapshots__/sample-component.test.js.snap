// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sample component renders without errors 1`] = `
<div>
  <p
    className="root"
  >
    Hello from component
  </p>
  <button
    onClick={undefined}
    type="button"
  >
    Click here
  </button>
</div>
`;
